zh/core-libraries/xml.rst

```diff
@@ -5,17 +5,155 @@ Xml
 
 .. php:class:: Xml
 
+``class Cake\Utility\Xml``
+
+Xml类充许将数组转换成 ``SimpleXMLElement`` 或者 ``DOMDocument`` 对象，并且可以转回数组。
+
+创建XML对象
+===========
+
+.. static Cake\Utility\Xml::build($input, array $options =[])
```

Comment:
`.. php:staticmethod::` should be used instead of `.. static` (like in english version)